app/assets/stylesheets/student_index.scss

```diff
@@ -0,0 +1,54 @@
+@import 'color';
+
+.student_index_container {
+  margin-top: 40px;
+
+  .student_block {
+    margin-right: 10px;
+    margin-bottom: 20px;
+    height: 120px;
+    border-style: solid;
+    border-color: $rock;
+    border-width: 1px;
+
+    background-color: $gray;
+
+    .link_student a {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 4<br>Selector `link_student` should be written in lowercase with hyphens